modules/apps/portal-search-solr8/portal-search-solr8-impl/src/main/java/com/liferay/portal/search/solr8/internal/connection/SolrClientManager.java

```diff
@@ -123,8 +112,6 @@ protected void setCloudSolrClientFactory(
 
 	@Reference(
 		cardinality = ReferenceCardinality.MULTIPLE,
-		policy = ReferencePolicy.DYNAMIC,
```

Comment:
@slnn Basically looks good to me, but please don't remove the DYNAMIC GREEDY from MULTIPLE references. We just want to make MANDATORY references static.

Reply:
@slnn I mean in this commit.

Reply:
Done